src/Controllers/ExampleController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Controllers;
+
+
+use App\Models\ExampleModel;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Slim\Http\Response;
+
+class ExampleController
```

Comment:
maybe we can change that to `ExampleApiRouteController` or something like that, to make it a little bit more realistic? again along the lines of my other comment, it would be good for the examples to be realistic because otherwise my students get confused about what controllers and models to make

Reply:
yeh i agree with the controller name. matches my suggestion of having it at another route too.

Reply:
Just to be super clear I'm not suggesting that we actually add these examples to the skeleton, I think it should be almost unchanged from it's current state, these Example things are just examples for those of you that haven't seen auto wiring before